kitsune/settings.py

```diff
@@ -1209,3 +1209,4 @@ def filter_exceptions(event, hint):
 TOLL_FREE_REGEX = re.compile(r'^.*8(00|33|44|55|66|77|88)[0-9]\d{6,}$')
 SIMPLE_DOMAIN_IP_REGEX = re.compile(r'(?:[^\s/]+\.[^\s0-9/.]{2,})|(?:\d{1,3}\.){3}\d{1,3}')
 REGEX_TIMEOUT = config('REGEX_TIMEOUT', default=5, cast=int)
+VANITY_PHONE_NUMBER_REGEX = re.compile(r'[0-9]{3}-?[a-zA-Z2-9][a-zA-Z0-9]{2}-?[a-zA-Z0-9]{4}')
```

Comment:
Strictly speaking this regex will match any well-formed number in the NANP, so perhaps this should be called "NANP_REGEX"